app/src/main/res/layout/wallet_detail_balance_layout.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    >
+  <RelativeLayout
```

Comment:
Same as before. Adding multiple relatives layouts makes the view slower and more complex. why not add constarints?